### DISCUSSION TITLE
https://thanhle.blog/vi/blog/react-18-co-gi-hot

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#20] [Ideas]
# https://thanhle.blog/vi/blog/react-18-co-gi-hot

https://thanhle.blog/vi/blog/react-18-co-gi-hot

**Comment by nguyenhoaikhang37** [comment]
một bài viết bổ ích quá anh ạ, mà bài này hình như thiên về kĩ thuật nên thấy anh viết nghiêm túc quá, em k quen kkk =]]]

**Reply by thanhlmm** [reply]
Cái này hơi khó, anh ko biết nêm nếm sao cho mặn e ơi :D
